### PR TITLE
GEODE-3048: Editing tests requiring Gradle test runner to fail descriptively in IntelliJ

### PR DESCRIPTION
**Pull request by YehEmily**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by YehEmily** [comment]
Precheckin in progress!

File: geode-assembly/src/test/java/org/apache/geode/rest/internal/web/controllers/RestAPIOnRegionFunctionExecutionDUnitTest.java [hunk 1]
```diff
@@ -26,7 +26,9 @@
 import java.util.Map;
 import java.util.Set;
 
+import org.apache.geode.test.dunit.rules.RequiresGeodeHome;
```
Comment (pdxrunner) [inline geode-assembly/src/test/java/org/apache/geode/rest/internal/web/controllers/RestAPIOnRegionFunctionExecutionDUnitTest.java]: Reorder imports - looks like there are several files that should have the imports reordered. I'd suggest reviewing all of the tests modified and use the IDE to reorder the imports.

File: geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommandsIntegrationTest.java [hunk 2]
```diff
@@ -39,6 +39,8 @@
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.ClassRule;
```
Comment (pdxrunner) [inline geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommandsIntegrationTest.java]: Imports should be reordered to the project standard ordering.

Comment (YehEmily) [inline geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommandsIntegrationTest.java]: Okay, I'll get that done and update the PR! Thank you!

**Pull request closed without merging**